book/src/jsonrpc-api.md

```diff
@@ -201,6 +202,25 @@ curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","id":1, "m
 {"jsonrpc":"2.0","result":268,"id":1}
 ```
 
+---
+### getConfirmation
+Returns the current cluster confirmation time in milliseconds
```

Comment:
current? How about "Returns the mean confirmation time (in milliseconds) within the last X seconds" where X is some constant.

Reply:
But it doesn't actually do that